htdocs/societe/list.php

```diff
@@ -821,6 +830,16 @@
 	print $form->selectarray("search_type_thirdparty", $formcompany->typent_array(0), $search_type_thirdparty, 0, 0, 0, '', 0, 0, 0, (empty($conf->global->SOCIETE_SORT_ON_TYPEENT) ? 'ASC' : $conf->global->SOCIETE_SORT_ON_TYPEENT), 'maxwidth75', 1);
 	print '</td>';
 }
+// Multiprice level
+if (!empty($conf->global->PRODUIT_MULTIPRICES) || !empty($conf->global->PRODUIT_CUSTOMER_PRICES_BY_QTY_MULTIPRICES))
```

Comment:
The test here should be useless. Because if false, $arrayfields['s.price_level']['checked'] will always be empty. Right ?

Reply:
> The test here should be useless. Because if false, $arrayfields['s.price_level']['checked'] will always be empty. Right ?

Reply:
@eldy I have a not understand to the first read on my phone, i resolved in the comit.